docs/en/changelogs/2.3.0.md

```diff
@@ -21,7 +21,7 @@ Basic auth isn't enabled by default on test sites.  If you need this, put this i
 ### /silverstripe and /cms no longer work as aliases to /admin
 
 Removed certain URL aliases for CMS interface to allow for common page URLs like "silverstripe" or "cms". Please use
-/admin as the main URL to access the CMS. See http://open.silverstripe.com/ticket/3267
+/admin as the main URL to access the CMS. See `http://open.silverstripe.com/ticket/3267`
```

Comment:
Looking at how GH parses markdown, this stops this being linked automatically.

I'm not sure there's much value wrapping these issues in backticks